crates/driver/src/infra/config/file/load.rs

```diff
@@ -264,11 +264,14 @@ pub async fn load(network: &blockchain::Network, path: &Path) -> infra::Config {
                     config.submission.retry_interval_secs,
                 ),
                 kind: match mempool {
-                    file::Mempool::Public { revert_protection } => {
-                        mempool::Kind::Public(if *revert_protection {
-                            mempool::RevertProtection::Enabled
-                        } else {
+                    file::Mempool::Public => {
+                        // If there is only one mempool - public mempool - revert protection is
+                        // disabled, otherwise driver would not even try to settle revertable
+                        // settlements
+                        mempool::Kind::Public(if config.submission.mempools.len() == 1 {
```

Comment:
I think it's possible to configure multiple public mempools (if e.g. you want to decentralize submission nodes geographically)

```suggestion
                        // If there is no private mempool, revert protection is
                        // disabled, otherwise driver would not even try to settle revertable
                        // settlements
                        mempool::Kind::Public(
                            if config
                                .submission
                                .mempools
                                .iter()
                                .any(|pool| matches!(pool, file::Mempool::MevBlocker { .. }))
```

Reply:
Fixed.